src/Questions.jsx

```diff
@@ -62,6 +104,8 @@ question: "In past times, what would a gentleman keep in his fob pocket?"
                                 key={index}
                                 id={uniqueAnsId}
                                 onClick={() => handleSelect(question.id, uniqueAnsId)}
+                                // This will not work unfortunately, as you will always get back a unique ID
+                                // and you the following logical operation will never be true: selectedAnswers[question.id] === uniqueAnsId
```

Comment:
need help for tracking and checking the answers if they are correct or not with state